libethereum/State.cpp

```diff
@@ -21,13 +21,13 @@
 
 #include "State.h"
 
-#include <boost/filesystem.hpp>
-#include <boost/timer.hpp>
 #include <libdevcore/Assertions.h>
 #include <libdevcore/TrieHash.h>
 #include <libevm/VMFactory.h>
-#include "BlockChain.h"
+#include <boost/filesystem.hpp>
```

Comment:
Includes sorting can also be turned off for now, be I believe we can config what we want with set of regex and priorities. I just didn't have time to do it.